tgstation.dme

```diff
@@ -1,3137 +1,3139 @@
-#include "code\modules\antagonists\abductor\equipment\glands\transform.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\trauma.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\ventcrawl.dm"
-#include "code\modules\antagonists\abductor\equipment\glands\viral.dm"
-#include "code\modules\antagonists\abductor\machinery\camera.dm"
-#include "code\modules\antagonists\abductor\machinery\console.dm"
-#include "code\modules\antagonists\abductor\machinery\dispenser.dm"
-#include "code\modules\antagonists\abductor\machinery\experiment.dm"
-#include "code\modules\antagonists\abductor\machinery\pad.dm"
-#include "code\modules\antagonists\blob\blob.dm"
-#include "code\modules\antagonists\blob\blob\blob_report.dm"
-#include "code\modules\antagonists\blob\blob\overmind.dm"
-#include "code\modules\antagonists\blob\blob\powers.dm"
-#include "code\modules\antagonists\blob\blob\theblob.dm"
-#include "code\modules\antagonists\blob\blob\blobs\blob_mobs.dm"
-#include "code\modules\antagonists\blob\blob\blobs\core.dm"
-#include "code\modules\antagonists\blob\blob\blobs\factory.dm"
-#include "code\modules\antagonists\blob\blob\blobs\node.dm"
-#include "code\modules\antagonists\blob\blob\blobs\resource.dm"
-#include "code\modules\antagonists\blob\blob\blobs\shield.dm"
-#include "code\modules\antagonists\blood_contract\blood_contract.dm"
-#include "code\modules\antagonists\brainwashing\brainwashing.dm"
-#include "code\modules\antagonists\brother\brother.dm"
-#include "code\modules\antagonists\changeling\cellular_emporium.dm"
-#include "code\modules\antagonists\changeling\changeling.dm"
-#include "code\modules\antagonists\changeling\changeling_power.dm"
-#include "code\modules\antagonists\changeling\powers\absorb.dm"
-#include "code\modules\antagonists\changeling\powers\adrenaline.dm"
-#include "code\modules\antagonists\changeling\powers\augmented_eyesight.dm"
-#include "code\modules\antagonists\changeling\powers\biodegrade.dm"
-#include "code\modules\antagonists\changeling\powers\chameleon_skin.dm"
-#include "code\modules\antagonists\changeling\powers\digitalcamo.dm"
-#include "code\modules\antagonists\changeling\powers\fakedeath.dm"
-#include "code\modules\antagonists\changeling\powers\fleshmend.dm"
-#include "code\modules\antagonists\changeling\powers\headcrab.dm"
-#include "code\modules\antagonists\changeling\powers\hivemind.dm"
-#include "code\modules\antagonists\changeling\powers\humanform.dm"
-#include "code\modules\antagonists\changeling\powers\lesserform.dm"
-#include "code\modules\antagonists\changeling\powers\linglink.dm"
-#include "code\modules\antagonists\changeling\powers\mimic_voice.dm"
-#include "code\modules\antagonists\changeling\powers\mutations.dm"
-#include "code\modules\antagonists\changeling\powers\panacea.dm"
-#include "code\modules\antagonists\changeling\powers\pheromone_receptors.dm"
-#include "code\modules\antagonists\changeling\powers\regenerate.dm"
-#include "code\modules\antagonists\changeling\powers
```

Comment:
The reason why travis failed was this file got rewritten as CRLF instead of LF. Please set the file type to LF in the bottom right corner of VSCode